recipes/spades/3.11.1/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: 3ab85d86bf7d595bd8adf11c971f5d258bbbd2574b7c1703b16d6639a725b474  # [linux]
 
 build:
-  number: 2
+  number: 3
```

Comment:
why the bump?

Reply:
@epruesse it fixed an error during circleci ("17:40:32 BIOCONDA ERROR BUILD ERROR: packages with divergent build strings in repository for recipe recipes/spades/3.11.1. A build number bump is likely needed: spades-3.11.1-he4cf2ce_2") https://circleci.com/gh/bioconda/bioconda-recipes/35487?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link